source/tutorials/tutorials-dashboard-apps/dashboard.rst

```diff
@@ -0,0 +1,142 @@
+.. _app-development-tutorials-dashboard-apps-dashboard:
+
+Developing the Dashboard App
+============================
+
+.. warning::
+    You muse have followed :ref:`enabling-development-mode` for any part of this tutorial to work.
+
+Strategy
+--------
+
+#. Start by completing the :ref:`enabling-development-mode` steps.
+#. Create a symlink named ``dev`` in the ``~/ondemand/misc/ondemand/<app>``. 
+#. In the app's symlinked directory, use ``git branch`` to start your development work.
+#. Create a ``.env.local`` file in the development dashboard app root to customize environment variables.
+#. Rebuild the ``dashboard`` app using ``bin/setup`` out of the app's root for configuration changes.
+#. Use the Sandbox to launch and iterate on your work.
+
+Create a Dev Dashboard
+----------------------
+#. ``clone`` the OOD repo into: ``cd ~/ondemand/misc`` then ``git clone git@github.com:OSC/ondemand.git``
+#. Work out of the ``dev`` directory/space: ``cd ~/ondemand/dev/``.
+#. Symlink to the ``dashboard`` app in the cloned repo: ``ln -s ../misc/ondemand/apps/dashboard/dashboard/ dashboard``
+#. Make a branch and to begin word on your dev dashboard: ``git checkout -b dev_work`` 
+#. Rebuild the ood dashboard app:  ``bin/setup``.
+#. Navigate to the Sandbox and launch your ``dashboard`` app to use the ``dev_work``.
+
+Notice the ``url`` for this ``dev_dashboard`` app. No longer do 
+you  see ``*/sys/dashboard/`` but instead ``*/dev/dashboard/``. 
```

Comment:
I think the `dev_dashboard` should be `dev/dashboard` no?  Plus - why replace `/pun` with `*` Can't we just say `/pun`?

```suggestion
Notice the ``url`` for this ``/pun/dev/dashboard`` app. No longer do 
you  see ``/pun/sys/dashboard/`` but instead ``/pun/dev/dashboard/``. 
```